The doc-tests seems to depend on lazy_regex which would form a dependency loop, so they are ignored.

diff --git a/mod.rs b/mod.rs
index ccaabc4..05e8222 100644
--- a/mod.rs
+++ b/mod.rs
@@ -37,14 +37,14 @@ where
 /// built at first use.
 ///
 /// Flags can be specified as suffix:
-/// ```
+/// ```ignore
 /// let case_insensitive_regex = regex!("^ab+$"i);
 /// ```
 ///
 /// The macro returns a reference to a [regex::Regex]
 /// or a [regex::bytes::Regex] instance,
 /// differentiated by the `B` flag:
-/// ```
+/// ```ignore
 /// let verbose = regex!(r#"_([\d\.]+)"#)
 ///     .replace("This is lazy-regex_2.2", " (version $1)");
 /// assert_eq!(verbose, "This is lazy-regex (version 2.2)");
@@ -58,7 +58,7 @@ pub fn regex(input: TokenStream) -> TokenStream {
 /// built at first use.
 ///
 /// Flags can be specified as suffix:
-/// ```
+/// ```ignore
 /// let case_insensitive_regex = bytes_regex!("^ab+$"i);
 /// assert!(case_insensitive_regex.is_match(b"abB"));
 /// ```
@@ -73,7 +73,7 @@ pub fn bytes_regex(input: TokenStream) -> TokenStream {
 ///
 /// Example:
 ///
-/// ```
+/// ```ignore
 /// pub static GLOBAL_REX: Lazy<Regex> = lazy_regex!("^ab+$"i);
 /// ```
 ///
@@ -88,7 +88,7 @@ pub fn lazy_regex(input: TokenStream) -> TokenStream {
 ///
 /// Example:
 ///
-/// ```
+/// ```ignore
 /// pub static GLOBAL_REX: Lazy<bytes::Regex> = bytes_lazy_regex!("^ab+$"i);
 /// ```
 ///
@@ -103,7 +103,7 @@ pub fn bytes_lazy_regex(input: TokenStream) -> TokenStream {
 /// time)
 ///
 /// Example:
-/// ```
+/// ```ignore
 /// let b = regex_is_match!("[ab]+", "car");
 /// assert_eq!(b, true);
 /// ```
@@ -123,7 +123,7 @@ pub fn regex_is_match(input: TokenStream) -> TokenStream {
 /// at compile time)
 ///
 /// Example:
-/// ```
+/// ```ignore
 /// let b = bytes_regex_is_match!("[ab]+", b"car");
 /// assert_eq!(b, true);
 /// ```
@@ -142,7 +142,7 @@ pub fn bytes_regex_is_match(input: TokenStream) -> TokenStream {
 /// second argument, as a `&str`, or a `&[u8]` if the `B` flag is set.
 ///
 /// Example:
-/// ```
+/// ```ignore
 /// let f_word = regex_find!(r#"\bf\w+\b"#, "The fox jumps.");
 /// assert_eq!(f_word, Some("fox"));
 /// let f_word = regex_find!(r#"\bf\w+\b"#B, "The forest is silent.");
@@ -167,7 +167,7 @@ pub fn regex_find(input: TokenStream) -> TokenStream {
 /// second argument as a `&[u8]`
 ///
 /// Example:
-/// ```
+/// ```ignore
 /// let f_word = bytes_regex_find!(r#"\bf\w+\b"#, b"The fox jumps.");
 /// assert_eq!(f_word, Some("fox".as_bytes()));
 /// ```
@@ -194,7 +194,7 @@ pub fn bytes_regex_find(input: TokenStream) -> TokenStream {
 /// will contain `""` instead.
 ///
 /// Example:
-/// ```
+/// ```ignore
 /// let (whole, name, version) = regex_captures!(
 ///     r#"(\w+)-([0-9.]+)"#, // a literal regex
 ///     "This is lazy_regex-2.0!", // any expression
@@ -231,7 +231,7 @@ pub fn regex_captures(input: TokenStream) -> TokenStream {
 /// will contain `b""` instead.
 ///
 /// Example:
-/// ```
+/// ```ignore
 /// let (whole, name, version) = bytes_regex_captures!(
 ///     r#"(\w+)-([0-9.]+)"#, // a literal regex
 ///     b"This is lazy_regex-2.0!", // any expression
@@ -353,7 +353,7 @@ fn bytes_replacen(input: TokenStream, limit: usize) -> TokenStream {
 /// Any optional group with no value is replaced with `""`.
 ///
 /// Example:
-/// ```
+/// ```ignore
 /// let text = "Fuu fuuu";
 /// let text = regex_replace!(
 ///     "f(u*)"i,
@@ -376,7 +376,7 @@ pub fn regex_replace(input: TokenStream) -> TokenStream {
 /// Any optional group with no value is replaced with `b""`.
 ///
 /// Example:
-/// ```
+/// ```ignore
 /// println!("{:?}", "ck ck".as_bytes());
 /// let text = b"Fuu fuuu";
 /// let text = bytes_regex_replace!(
@@ -400,7 +400,7 @@ pub fn bytes_regex_replace(input: TokenStream) -> TokenStream {
 /// Any optional group with no value is replaced with `""`.
 ///
 /// Example:
-/// ```
+/// ```ignore
 /// let text = "Foo fuu";
 /// let text = regex_replace_all!(
 ///     r#"\bf(?P<suffix>\w+)"#i,
@@ -423,7 +423,7 @@ pub fn regex_replace_all(input: TokenStream) -> TokenStream {
 /// Any optional group with no value is replaced with `""`.
 ///
 /// Example:
-/// ```
+/// ```ignore
 /// let text = b"Foo fuu";
 /// let text = bytes_regex_replace_all!(
 ///     r#"\bf(?P<suffix>\w+)"#i,
